test/jaxp/javax/xml/jaxp/unittest/validation/ErrorHandlingTest.java

```diff
@@ -0,0 +1,126 @@
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package validation;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.ArrayList;
+import static java.util.Collections.unmodifiableList;
+import java.util.List;
+import java.util.Locale;
+import static java.util.Locale.ENGLISH;
+import javax.xml.XMLConstants;
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.xml.sax.ErrorHandler;
+import org.xml.sax.SAXException;
+import org.xml.sax.SAXParseException;
+
+/*
+ * @test
+ * @bug 8298087
+ * @library /javax/xml/jaxp/libs /javax/xml/jaxp/unittest
+ * @run testng validation.ErrorHandlingTest
+ * @summary Verifies error handling.
+ */
+public class ErrorHandlingTest {
+    private final static String xsd = "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>"
+            +"<schema xmlns=\"http://www.w3.org/2001/XMLSchema\""
+            +"       targetNamespace=\"https://errorhandlingtest/schema/Example\""
+            +"        elementFormDefault=\"qualified\">"
+            +"\n"
+            +"    <element name=\"root\">"
+            +"        <complexType>"
+            +"            <sequence>"
+            +"                <element name=\"a\">"
+            +"                    <complexType>"
+            +"                        <simpleContent>"
+            +"                            <extension base=\"string\">"
+            +"                                <attribute name=\"enabled\" type=\"boolean\" use=\"required\"/>"
+            +"                            </extension>"
+            +"                        </simpleContent>"
+            +"                    </complexType>"
+            +"                </element>"
+            +"            </sequence>"
+            +"        </complexType>"
+            +"    </element>"
+            +"\n"
+            +"</schema>";
+
+    private final static String xml = "<e:root xmlns:e=\"https://errorhandlingtest/schema/Example\">\n"
+            +"    <e:a>string</e:a>\n"
+            +"</e:root>";
+
+    /**
+     * Verifies that validation error is reported properly (once rather than twice).
+     *
+     * @throws Exception if the test fails
+     */
+    @Test
+    public void test() throws Exception {
+        Locale.setDefault(ENGLISH);
```

Comment:
I believe this test is run in `othervm` mode, but still, it would be preferred to restore the default locale after the test

Reply:
Thanks!  The line is removed.